EIPS/eip-1620.md

```diff
@@ -1,296 +1,185 @@
 ---
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg)
+author: Paul Razvan Berg (@PaulRBerg)
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Last Call
 type: Standards Track
 category: ERC
 created: 2018-11-24
+requires: 20
+review-period-end: 2020-09-24
 ---
 
 ## Simple Summary
-Money streaming represents the idea of continuous payments over a finite period of time. Block numbers are used as a proxy of time to continuously update balances.
+
+We define money streaming as continuous payments over a finite period of time. Block timestamps are used as an on-chain proxy for time.
 
 ## Abstract
-The following describes a standard whereby time is measured using block numbers and streams are mappings in a master contract.
 
-1. A provider sets up a money streaming contract.
-2. A prospective payer can interact with the contract and start the stream right away by depositing the funds required for the chosen period.
-3. The payee is able to withdraw money from the contract based on its ongoing solvency. That is: `payment rate * (current block height - starting block height)`
-4. The stream terms (payment rate, length, metadata) can be updated at any time if both parties pledge their signatures.
-5. The stream can be stopped at any point in time by any party without on-chain consensus.
-6. If the stream period ended and it was not previously stopped by any party, the payee is entitled to withdraw all the deposited funds.
+The following describes a standard whereby time is measured using block timestamps and streams are mappings in a master contract.
+
+1. Developer sets up a money streaming contract.
+2. A payer, henceforth referred to as sender, can create a stream by depositing funds in the contract and choosing a streaming duration.
+3. A payee, henceforth referred to as recipient, can withdraw money from the stream based on its continuous solvency. That is, `payment * (current block timestamp - starting block timestamp)`.
+4. The stream can be terminated by either the sender or the recipient.
+5. If the stream ended and has not been terminated, the recipient is entitled to withdraw all the remaining balance.
 
 ## Motivation
-This standardised interface aims to change the way we think about long-term financial commitments. Thanks to blockchains, payments need not be sent in chunks (e.g. monthly salaries), as there is much less overhead in paying-as-you-go. Money as a function of time would better align incentives in a host of scenarios.
+
+This standardised interface takes a stab at changing the way we think about trust in financial contracts. Thanks to blockchains, payments need not be sent in lumps (as with monthly salaries), as there is less overhead in pay-as-you-go. Money as a function of time would better align incentives in a host of scenarios.
```

Comment:
Recommend phrasing this from the point of view of a final EIP.  This means that this isn't "taking a stab at" anything, but instead it is strongly defining something.

Reply:
+1. If it is "taking a stab at" this implies the EIP is not functional. Should be rephrased.